modules/ROOT/pages/configure-local-registry.adoc

```diff
@@ -124,3 +190,49 @@ This command ensures that Runtime Fabric creates the necessary overrides for the
 The `rtfctl uninstall` command deletes any pull secret you specify during installation.
 
 When using `rtfctl` to perform a backup and restore procedure, the backups point to your local registry URL and pull secret. 
+--
+ 
+Helm::
++
+--
+. <<synchronize,Synchronize your container images>> to your local registry.
+. xref:install-helm.adoc#create-a-runtime-fabric-using-runtime-manager[Create a Runtime Fabric using Runtime Manager], and gather the activation data.
+. In Runtime Manager, select the *Helm* path, and follow the instructions on the screen platform.
+. Obtain your private `docker-server`, `docker-username`, and `docker-password`.
+. Following the *Helm* path instructions in Runtime Manager, create a secret with your docker server name, username, and password by running the following command in your Unix shell:
++
+[source,copy]
+----
+kubectl create secret docker-registry rtf-pull-secret --namespace test-gke-rtf-qax-helm --docker-server=<docker-servername> --docker-username=<docker-username --docker-password=<docker-password> --docker-email=<docker-email>
+----
+
+[start=6]
+. Following the *Helm* path instructions, download the values.yaml file and modify its values. The following is a values.yaml file example:
++
+[source,copy]
+----
+activationData: xxx
+proxy:
+ http_proxy:
+ http_no_proxy:
+ monitoring_proxy:
+muleLicense: 
+customLog4jEnabled: false
+global:
+ crds:
+  install: false
+ authorizedNamespaces: false
+ image:
+  rtfRegistry: 
```

Comment:
Maybe copying the whole file here makes it loose focus on what is specific to using the private registry. What they need to configure is this rtfRegistry and the pullSecretName